custom-namespaces.yaml

```
@@ -837,6 +841,22 @@ firefox_ios:
       type: table_explore
       views:
         base_view: clients_activation
+focus_ios:
+  owners:
+    - loines@mozilla.com
```

I'd recommend adding a DS and a DE in each, or even better a Github team.
```suggestion
    - loines@mozilla.com
    - kik@mozilla.com
```